tasks/managed/populate-release-notes/tests/test-populate-release-notes-non-rhsa-references.yaml

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-populate-release-notes-non-rhsa-references
+spec:
+  description: |
+    Run the populate-release-notes task with a type that is RHSA. Ensure that no additional references
+    are added
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > "$(workspaces.data.path)/data.json" << EOF
+              {
+                "releaseNotes": {
+                  "product_id": 123,
+                  "product_name": "Red Hat Openstack Product",
+                  "product_version": "123",
+                  "cpe": "cpe:/a:example:openstack:el8",
+                  "type": "RHBA",
+                  "issues": {
+                    "fixed": [
+                      {
+                        "id": "RHOSP-12345",
+                        "source": "issues.example.com"
+                      },
+                      {
+                        "id": 1234567,
+                        "source": "bugzilla.example.com"
+                      }
+                    ]
+                  },
+                  "synopsis": "test synopsis",
+                  "topic": "test topic",
+                  "description": "test description",
+                  "solution": "test solution",
+                  "references": []
```

Comment:
This line should be removed now that the schema was changed to not require references. At first I was like: Why didn't something fail with this? But there is no schema validation here, so that's why, right?

Reply:
And the check in `check-result` should be modified to check `references` is not there. Something like:
```
test "$(jq '.releaseNotes | has("references")' "$(workspaces.data.path)/data.json")" == "false"
```

Reply:
Good thinking. Fixed